package.json

```diff
@@ -15,7 +17,9 @@
     "request": "^2.87.0"
   },
   "devDependencies": {
-    "hubot": "^3.3.2"
```

Comment:
Are we going to bump the version number here?

Reply:
Which version #? I updated the hubot-google-hahgouts-chat version to 3.0.0.  Are you talking about the hubot version?

Reply:
I updated the hubot version to the latest version of hubot.